store/postgres/src/subgraphs.graphql

```diff
@@ -54,6 +54,7 @@ type EthereumContractDataSource @entity {
     source: EthereumContractSource!
     mapping: EthereumContractMapping!
     templates: [EthereumContractDataSourceTemplate!]
+    context: String # arbitrary entity serialized to JSON
```

Comment:
`JSON string representing a map with string keys and arbitrary values` perhaps?

Also, let's add this description above the field, like so:
```
"JSON string representing a map with string keys and arbitrary values"
context: String
```
That way it'll show up in the introspection :)

Reply:
I've noticed there's no reason for this field to be on `EthereumContractDataSource`, it should be only in `DynamicEthereumContractDataSource`.

To your point, I should put it as a field description. I made this:
```
"JSON object with user-provided keys and values, can be parsed as an entity"
```
What do you think?

Reply:
Perhaps `like` an entity instead of `as`?